libs/secrets/src/aes.cpp

```diff
@@ -41,8 +40,9 @@ vector<uint8_t> AES_CBC::encrypt(std::string_view input) {
     int c_len{0};
     int f_len{0};
 
-    ConcordAssert(OPENSSL_SUCCESS ==
-                  EVP_EncryptInit_ex(ctx.get(), EVP_aes_256_cbc(), nullptr, params_.key.data(), params_.iv.data()));
+    ConcordAssert(
+        OPENSSL_SUCCESS ==
+        EVP_EncryptInit_ex(ctx.get(), EVP_aes_256_cbc(), nullptr, getKeyParams().key.data(), getKeyParams().iv.data()));
```

Comment:
why is it using aes_256_cbc?

Reply:
Since we are supporting both (CBC and GCM) and this method is CBC implementation.